### DISCUSSION TITLE
Welcome to node-express-mongoose-boilerplate Discussions!

### DISCUSSION THREAD

**Discussion by hagopj13** [hagopj13/node-express-boilerplate#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by SouariKarim** [comment]


**Reply by hagopj13** [reply]
@SouariKarim do you also find the `README.md` file difficult to read?
